rs/bitcoin/ckbtc/minter/tests/replay_events.rs

```diff
@@ -199,7 +202,13 @@ impl GetEventsFile {
         let mut decompressed_buffer = Vec::new();
         gz.read_to_end(&mut decompressed_buffer)
             .expect("BUG: failed to decompress events");
-        Decode!(&decompressed_buffer, GetEventsResult).expect("Failed to decode events")
+        // todo() The logic here assumes the compressed events in the file still use the 'old'
```

Comment:
nit: Can you add a clean-up ticket (in the [maintenance epic](https://dfinity.atlassian.net/browse/XC-147)) and reference it here and below so that we don't forget about it?

Reply:
Done: [XC-261](https://dfinity.atlassian.net/browse/XC-261).